src/app/about/about.component.html

```diff
@@ -1,91 +1,189 @@
+            example.
+            Evaluating
+            them on other types of software is insubstantial. When mining GitHub, the scope of the collection can
+            be
+            restricted
+            to the target of the analysis/tool, and provide an adapted, yet still representative collection of
+            up-to-date
+            applications.
+            The current collection targets Java web applications, and the same automated mining methodology can be
+            applied to
+            different
+            types of target programs.</li>
+        </ul>
+        <p>For comparison and replication purposes, old versions of the collections are archived.</p>
+      </mat-expansion-panel>
+      <mat-expansion-panel>
+        <mat-expansion-panel-header>
+          How can I create a collection?
+        </mat-expansion-panel-header>
+        <p>To create a collection you need to be loggedin, then search for the project type you want in your
+          collection,
+          add as many project as you want then click the create collection button.
+        </p>
+      </mat-expansion-panel>
+      <mat-expansion-panel>
+        <mat-expansion-panel-header>
+          How can I download a collection?
+        </mat-expansion-panel-header>
+        <p>Collections that are made public by their owners are published on <a href="https://zenodo.org">Zenodo</a>.
+          If you have the DOI, you can search for it directly on Zenodo.</p>
+        <p>Alternatively, you can also browse collections on the
+          <a href="#/">Collections</a> page.</p>
+      </mat-expansion-panel>
+      <mat-expansion-panel>
+        <mat-expansion-panel-header>
+          How can I contribute?
+        </mat-expansion-panel-header>
+        <p>With the ABM methodology, we strive to create more dynamic, easily updatable benchmark suites that are
+          adapted
+          to
+          the needs
+          of different target domains. The overhead of this semi-automated methodology is the manual creation of
+          the
+          different
+          filters, which are written on top of the
+          <a href="http://www.opal-project.de/Hermes.html">Hermes</a> framework. We are always happy to add new
+          filters.
+          To
+          submit your own Hermes filters, please contact us by
+          email.
+        </p>
+        <p>Alongside, our open-source git repository can be found
+          <a href="https://github.com/ABenchM/abm">
+            here.</a> Contributions are welcomed!</p>
 
+      </mat-expansion-panel>
+      <mat-expansion-panel>
+        <mat-expansion-panel-header>
+          What is the source of the projects used in ABM
+        </mat-expansion-panel-header>
+        <p>We currently crawl open-source public repository from the following platforms for codes that match the
+          projects based on user filters</p>
+        <ul>
+          <li>Github</li>
+          <li>Maven</li>
```

Comment:
Only Maven for now. Can you also put a link to Delphi (https://github.com/delphi-hub/)?